### PR TITLE
Priojeet/update: change hyperlinked account details for del and dmx

### PR DESCRIPTION
**Pull request by priojeet-deriv**
# Description

Summary

Changes:

-   ...

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/regulatory/index.js [hunk 1]
```diff
@@ -189,11 +185,9 @@ const Regulatory = () => {
                 </SmallContainer>
             </SectionContainer>
             <SectionContainer padding="0 0 4rem 0">
```
Comment (AbdallaMourad) [inline src/pages/regulatory/index.js:187]:
```suggestion
            <SectionContainer padding="0 0 4rem">
```

**Review by AbdallaMourad — DISMISSED** [review]

File: src/pages/regulatory/index.js (hunk 1 above)
Comment (priojeet-deriv) [reply src/pages/regulatory/index.js:187]: done.

**Review by yashim-deriv — DISMISSED** [review]

File: src/components/localization/localized-link.js [hunk 2]
```diff
@@ -139,7 +139,9 @@ export const LocalizedLink = React.forwardRef(({ to, ...props }, ref) => {
                     target={target}
                     rel={rel}
                     className={className}
-                    style={style}
+                    style={
+                        style ? style : { color: `var(--color-black-3)`, textDecoration: `none` }
```
Comment (AbdallaMourad) [inline src/components/localization/localized-link.js:143]: If the value of `style` can have one of these values an `object` or `undefined` we can replace this syntax with `style={style ?? {color:...}}`

File: src/pages/regulatory/index.js [hunk 3]
```diff
@@ -206,92 +200,90 @@ const Regulatory = () => {
...
+                                    )}
+                                </Text>
+                            </CssGridColumn>
+                            <ResponsiveGrid align="flex-start">
+                                <Mobile>
+                                    <StyledHeader as="h4" type="sub-section-title" mb="1.6rem">
+                                        {localize('Deriv (MX) Ltd')}
+                                    </StyledHeader>
+                                </Mobile>
+                                <Img src={IOM} alt="IOM" />
```
Comment (AbdallaMourad) [inline src/pages/regulatory/index.js:251]: Can we use an `alt` that is more descriptive 🙏🏻

**Review by suthesan-deriv — DISMISSED** [review]

File: src/components/localization/localized-link.js (hunk 2 above)
Comment (priojeet-deriv) [reply src/components/localization/localized-link.js]: nice one. didn't know about this operator. done.

File: src/pages/regulatory/index.js (hunk 3 above)
Comment (priojeet-deriv) [reply src/pages/regulatory/index.js]: sure. the code was added before. done.

**Review by mustofa-binary — CHANGES_REQUESTED** [review]

File: src/pages/regulatory/index.js [hunk 4]
```diff
@@ -188,12 +184,10 @@ const Regulatory = () => {
...
                 </SmallContainer>
             </SectionContainer>
-            <SectionContainer padding="0 0 4rem 0">
-                <Eu>
-                    <SmallContainer fd="column">
-                        <DocumentAccordion />
-                    </SmallContainer>
-                </Eu>
+            <SectionContainer padding="0 0 4rem">
+                <SmallContainer fd="column">
```
[the remark this answers is not in the document]
Comment (priojeet-deriv) [reply src/pages/regulatory/index.js:188]: yeah. according to Islam's comment on redmine issue (#15248), "Basically this page should be exactly the same between EU IPs and Non-EU IPs EXCEPT for The Financial Commission section. That should stay as is.". Also, the footer will be different for EU and non-EU.

**Review by mustofa-binary — APPROVED** [review]

**Pull request merged**